src/components/directory-content.js

```diff
@@ -719,60 +753,37 @@ const DirectoryContent = () => {
         setSelectedUuids(new Set());
     }, [handleError, currentChildren, currentChildrenRef]);
 
-    const contextualMixPolicies = {
-        BIG: 'GoogleMicrosoft', // if !selectedUuids.has(selected.Uuid) deselects selectedUuids
-        ZIMBRA: 'Zimbra', // if !selectedUuids.has(selected.Uuid) just use activeElement
-        ALL: 'All', // union of activeElement.Uuid and selectedUuids (actually implemented)
-    };
-    let contextualMixPolicy = contextualMixPolicies.ALL;
-
-    const getSelectedChildren = (mayChange = false) => {
-        let acc = [];
-        let ctxtUuid = activeElement ? activeElement.elementUuid : null;
-        if (activeElement) {
-            acc.push(
-                Object.assign(
-                    {
-                        subtype:
-                            childrenMetadata[activeElement.elementUuid]
-                                ?.subtype,
-                    },
-                    activeElement
-                )
-            );
-        }
-
-        if (selectedUuids && currentChildren) {
-            if (
-                contextualMixPolicy === contextualMixPolicies.ALL ||
-                ctxtUuid === null ||
-                selectedUuids.has(ctxtUuid)
-            ) {
-                acc = acc.concat(
+    const getSelectedChildren = () => {
+        let selectedChildren = [];
+        if (currentChildren?.length > 0) {
+            // Adds the previously selected elements
+            if (selectedUuids?.size) {
+                selectedChildren = selectedChildren.concat(
                     currentChildren
                         .filter(
                             (child) =>
                                 selectedUuids.has(child.elementUuid) &&
                                 child.elementUuid !== activeElement?.elementUuid
                         )
-                        .map((child2) => {
-                            return Object.assign(
-                                {
-                                    subtype:
-                                        childrenMetadata[child2.elementUuid],
-                                },
-                                child2
-                            );
+                        .map((child) => {
+                            return {
+                                subtype: childrenMetadata[child.elementUuid],
```

Comment:
Where `.subtype` ? seems like an existing bug to me ... We won't have a lot of subtypes in ContentContextualMenu or in ContentToolbar I think...

Reply:
Ok, added the missing `?.subtype`